### ISSUE TITLE
All links on homepage are broken [Firefox-based browsers]

### ISSUE THREAD

**Issue by Wave6677** [modrinth/code#2137]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)

### What browsers are you seeing the problem on?

Firefox, Other (please specify)

### Describe the bug

When visiting https://modrinth.com in a new tab (specifically modrinth.com, not modrinth.com/mods, /plugins etc.) and trying to click any link on the hope page, it does not refresh the page at all to show the content on the new page. It shows the url changing in the url bar but the content of the page does not change at all.

### Steps to reproduce

1. Go to https://modrinth.com
2. Click any link
3. See the homepage and not the link you just clicked

### Expected behavior

You get taken to the link you clicked

### Additional context

I initially thought it might've been because I use Librewolf, however after installing base Firefox via the fedora repos it still did not work.

**Comment by BlueStaggo** [comment]
I use Firefox and it works just fine for me ¯\\\_(ツ)\_/¯

**Comment by Handrail9** [comment]
> I use Firefox and it works just fine for me ¯\_(ツ)_/¯

Perhaps its only if the browser hasn't visited modrinth before/has no cookies from modrinth?

**Comment by linux-leo** [comment]
can confirm this issue on firefox and edge.

**Comment by Mari023** [comment]
same for me on firefox 115.14.0esr
it does work in a private window however
I also tried deleting all cookies and website data for modrinth, that also didn't help
what *did* help, was disabling the `I don't care about cookies` extension (which reminded me that I should replace it with `I still don't care about cookies`, tho that is also broken)
someone already opened an issue for I still don't care about cookies: https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/15323

TLDR: disable `I don't care about cookies` and / or `I still don't care about cookies`

**Comment by Prospector** [comment]
going to go ahead and close since it's 2 years old and caused by an extension
